MREUnityRuntime/MREUnityRuntimeLib/Core/RigidBody.cs

```diff
@@ -182,7 +182,8 @@ internal void ApplyPatch(RigidBodyPatch patch, bool patchVelocities)
 			}
 			if (patch.IsKinematic.HasValue)
 			{
-				_rigidbody.isKinematic = _rigidbody.isKinematic.GetPatchApplied(IsKinematic.ApplyPatch(patch.IsKinematic));
+				IsKinematic = patch.IsKinematic.Value;
+				//_rigidbody.isKinematic = _rigidbody.isKinematic.GetPatchApplied(IsKinematic.ApplyPatch(patch.IsKinematic));
```

Comment:
Why is this commented out?

Reply:
we handle that in physics bridge

---
In reply to: [448296417](https://github.com/microsoft/mixed-reality-extension-unity/pull/199#discussion_r448296417) [](ancestors = 448296417)

Reply:
I'll delete once I confirm few more things, but leave it commented for now

---
In reply to: [448337558](https://github.com/microsoft/mixed-reality-extension-unity/pull/199#discussion_r448337558) [](ancestors = 448337558,448296417)